packages/react-component-library/src/components/DatePicker/DatePicker.tsx

```diff
@@ -69,11 +69,11 @@ export const DatePicker: React.FC<DatePickerProps> = ({
   label = 'Select Date',
   onBlur,
   onChange,
-  disabled,
   placement = DATEPICKER_PLACEMENT.BELOW,
   startDate,
   endDate,
   isRange,
+  isDisabled = false,
```

Comment:
`isDisabled` will default to `false` if it is not set, ie it is not nullable

Reply:
Yeah I see it'll still be falsey i'll remove the default